addons/sitting/functions/fnc_sit.sqf

```diff
@@ -16,23 +16,14 @@
  */
 #include "script_component.hpp"
 
-private ["_actionID", "_configFile", "_sitDirection", "_sitPosition", "_sitRotation", "_seatPosOrig"];
+private ["_actionID", "_configFile", "_sitDirection", "_sitPosition", "_seatPosOrig"];
 
 params ["_seat", "_player"];
 
-// Set global variable for standing up
-GVAR(seat) = _seat;
-
 // Overwrite weird position, because Arma decides to set it differently based on current animation/stance...
 _player switchMove "amovpknlmstpsraswrfldnon";
 
 // Add scroll-wheel action to release object
-_actionID = _player getVariable [QGVAR(StandUpActionID), -1];
-
-if (_actionID != -1) then {
-    _player removeAction _actionID;
-};
```

Comment:
These lines are needed to prevent duplicate actions from being added. Could possibly happen in certain `selectPlayer` / multiple Zeus scenarious. I'd add that back in.